go.mod

```diff
@@ -1,18 +1,56 @@
 module github.com/mudler/docker-companion
 
-go 1.12
+go 1.24.3
```

Comment:
That might be a bit too bleeding edge for some (I only have access to 1.24.2's snap for example). Would it still work with a slightly older version? 1.18 or 1.20 maybe?